desc/vmec.py

```diff
@@ -208,7 +208,9 @@ def load(
         return eq
 
     @classmethod
-    def save(cls, eq, path, surfs=128, verbose=1):  # noqa: C901 - FIXME - simplify
+    def save(  # noqa: C901 - FIXME - simplify
+        cls, eq, path, surfs=128, verbose=1, M_nyq=None, N_nyq=None
```

Comment:
I won't force you to change this, but I would prefer to consistently have `verbose` always be the last kwarg

Reply:
I won't just because changing it means changing positional args that others might be using and that would break code. I know we have not committed to an API but for this I will keep as is

Reply:
maybe relevant: https://docs.python.org/3.8/whatsnew/3.8.html#positional-only-parameters

Reply:
This is cool! In a future PR I would be in favor of requiring stuff like `verbose` to be keyword arguments. That would help with backward-compatibility of the code